ES/ESmy028.xml

```diff
@@ -673,8 +673,8 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                                 <num value="4">፬</num>
```

Comment:
for quire 1 at line 652
instead of
   `<locus target="#1r"/> `
you should have
`<dim unit="leaf">2</dim>`